src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -295,11 +295,12 @@ def init_filesystem_fields(partition)
 
       # @param fs [Filesystem::BlkFilesystem]
       def init_mount_options(fs)
-        if fs.mountpoint && !fs.mountpoint.empty?
-          @mount = fs.mountpoint
-          @mountby = fs.mount_by.to_sym
+        if !fs.mount_point.nil?
+          @mount = fs.mount_point.path
+          @mountby = fs.mount_point.mount_by.to_sym
```

Comment:
Consistency: @mountby here vs. mount_by everywhere else in the code

Reply:
The same than above.

Reply:
OK